### PR TITLE
Use rustup 'minimal' distribution by default

### PR DESCRIPTION
**Pull request by woshilapin**
Thanks to @BanzaiMan who reviewed https://github.com/travis-ci/travis-build/pull/1799, I'm creating the current PR to modify the documentation about how the `rust` builder is made.

### CODE REVIEW

File: user/languages/rust.md [hunk 1]
```diff
@@ -33,8 +33,13 @@ new to Travis CI please read our [Tutorial](/user/tutorial/) and
 ## Choosing a Rust version
 
 By default, we download and install the latest stable Rust release at the start
-of the build, along with appropriate language tools including `cargo`, `rustc`,
-`rustdoc`, `rust-gdb`, `rust-lldb`, and `rustup`.
+of the build (thanks to `rustup`). The [`minimal` profile][profiles] is used
+and include the following language tools `cargo`, `rustc` and `rustup`.
```
Comment (Pezi777) [inline user/languages/rust.md]:
```suggestion
and includes the following language tools `cargo`, `rustc`, and `rustup`.
```

Comment (woshilapin) [reply user/languages/rust.md]: Thanks @Pezi777. What is the used practice on this repo: do I need to keep it as a new commit? Do I need to rebase and squash it?

Comment (Pezi777) [reply user/languages/rust.md]: hey @woshilapin just click on "Commit suggestion" > then I will merge it into master

Comment (slonopotamus) [reply user/languages/rust.md]: > then I will merge it into master

Please note that travis-ci/travis-build#1799 that implements this change was not yet merged.

**Comment by Pezi777** [comment]
@woshilapin thank you so much :-)

**Pull request merged**